Patch-Source: https://github.com/alsa-project/alsa-utils/pull/309
---
From f4c561b16ca6b07562e83b469c06d1cfb1af3038 Mon Sep 17 00:00:00 2001
From: Achill Gilgenast <achill@achill.org>
Date: Mon, 24 Nov 2025 15:53:04 +0100
Subject: [PATCH] configure: Allow systemd service installation without
 systemd.pc

If the systemdsystemunitdir got passed anyway as a variable, we don't
need the systemd.pc dependency anymore and allow building without a
systemd build-dependency.

Relevant in Alpine Linux, where we allowed systemd services to be
(sub-)packaged (e.g. for downstreams like postmarketOS), but don't have
systemd pacakged in Alpine itself, yet.

Signed-off-by: Achill Gilgenast <achill@achill.org>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 45327789..00a613cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -419,8 +419,8 @@ AC_ARG_WITH([systemdsystemunitdir],
 if test "x$with_systemdsystemunitdir" != xno; then
         AC_SUBST([systemdsystemunitdir], [$with_systemdsystemunitdir])
 fi
-AM_CONDITIONAL(HAVE_SYSTEMD, [test "$have_min_systemd" = "yes" \
-        -a -n "$with_systemdsystemunitdir" -a "x$with_systemdsystemunitdir" != xno ])
+AM_CONDITIONAL(HAVE_SYSTEMD, [test -n "$with_systemdsystemunitdir" \
+			      -a "x$with_systemdsystemunitdir" != xno ])
 
 AC_ARG_WITH([asound-state-dir],
         AS_HELP_STRING([--with-asound-state-dir=DIR], [Directory to place asound.state file in]),
